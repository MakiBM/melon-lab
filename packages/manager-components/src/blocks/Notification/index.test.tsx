import React from 'react';
import Notification from './index';

describe('Notification', () => {
  const defaultElement = <Notification />;

  it('should render correctly', () => {
    const wrapper = shallow(defaultElement);
    expect(wrapper).toMatchSnapshot();
  });
});
