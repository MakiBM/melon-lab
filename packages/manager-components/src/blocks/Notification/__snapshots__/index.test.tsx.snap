// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification should render correctly 1`] = `
<div
  className="jsx-__scopedHash notification"
>
  <JSXStyle
    css="__scoped"
    styleId="__scopedHash"
  />
</div>
`;
